tests/components/sensor/test_pilight.py

```diff
@@ -0,0 +1,120 @@
+"""The tests for the Pilight sensor platform."""
+import logging
+
+from homeassistant.bootstrap import setup_component
+import homeassistant.components.sensor as sensor
+from homeassistant.components import pilight
+
+from tests.common import get_test_home_assistant, assert_setup_component
+
+HASS = None
+
+
+def fire_pilight_message(protocol, data):
+    """Fire the fake pilight message."""
+    message = {pilight.ATTR_PROTOCOL: protocol}
+    message.update(data)
+    HASS.bus.fire(pilight.EVENT, message)
+
+
+def setUpModule():   # pylint: disable=invalid-name
+    """Initialize a Home Assistant server."""
+    global HASS
```

Comment:
Preferably use `self.hass = get_test_home_assistant()` here instead of `globals`